File: frontend/src/models/Vrf.ts
export default class Vrf {
    name!: string;
    vrf_type!: string;
    admin_state!: string;
    oper_state!: string;
    oper_down_reason!: string;

    constructor(jsonObj?: Vrf) {
        if (jsonObj) {
            this.name = jsonObj.name;
            this.vrf_type = jsonObj.vrf_type;
            this.admin_state = jsonObj.admin_state;
            this.oper_state = jsonObj.oper_state;
            this.oper_down_reason = jsonObj.oper_down_reason;
        }
    }
}

File: backend/apps/vrfs/vrf.py
class Vrf():
    def __init__(self, name, vrf_type, admin_state, oper_state, oper_down_reason):
        self.name = name
        self.vrf_type = vrf_type
        self.admin_state = admin_state
        self.oper_state = oper_state
        self.oper_down_reason = oper_down_reason